rllib/tuned_examples/bc/cartpole-bc.py

```diff
@@ -0,0 +1,91 @@
+from pathlib import Path
```

Comment:
nit: rename into `cartpole_bc.py` (underscore instead of hyphen). Main reason: hyphens make it impossible to import as python modules by other script.